### ISSUE TITLE
Option to authorize `eth_accounts` at injection

### ISSUE THREAD

**Issue by dahu33** [cawabunga/headless-web3-provider#17]
Currently, after injecting headless-web3-provider, the accounts are not returned by `eth_accounts` until  a `eth_requestAccounts` RPC call has been made and authorized.

While this behavior is perfect for testing a user visiting a dApp for the first time, it's less than ideal for cases when the user has already "connected" one or multiple accounts to the dApp (think MetaMask here).

It would be very useful if headless-web3-provider had an option to authorize `eth_accounts` at the time of injection, without requiring a `eth_requestAccounts` RPC call and calling `wallet.authorize(Web3RequestKind.RequestAccounts);`.

**Comment by cawabunga** [comment]
Hey @dahu33, that makes sense to closely emulate real wallet behavior. And thanks for describing the way you wanted to use it.

I have added this feature and published new version `0.2.4`.

Here is an example how to auto authorize `eth_accounts`:

```ts
import { test as base } from '@playwright/test'
import { injectHeadlessWeb3Provider } from 'headless-web3-provider'

export const test = base.extend({
  injectWeb3Provider: async (use) => {
    await use((page) =>
      injectHeadlessWeb3Provider(
        page,
        privateKeys,
        31337,
        'http://localhost:8545',
        { permitted: [Web3RequestKind.Accounts] } // <-- new config option
      )
    )
  },
})
```

**Comment by dahu33** [comment]
@cawabunga amazing, thanks for addressing this issue!
